### PR TITLE
Better release yamls

### PR DESCRIPTION
**Pull request by matzew**
Done for [SRVKE-595](https://issues.redhat.com/browse/SRVKE-595)

/assign @aliok 

On the commits, I gave description on what/why

### CODE REVIEW

File: openshift/release/resolve.sh [hunk 1]
```diff
@@ -20,6 +20,7 @@ function resolve_resources(){
     # 4. Remove empty lines
     sed -e "s+\(.* image: \)\(knative.dev\)\(.*/\)\(test/\)\(.*\)+\1\2 \3\4test-\5+g" \
         -e "s+ko://++" \
+        -e "s+knative-sources+knative-eventing+" \
```
Comment (aliok) [inline openshift/release/resolve.sh:23]: why is this necessary?

Comment (matzew) [reply openshift/release/resolve.sh:23]: it does replace the `knative-sources` references, with the `knative-eventing`. 

This commit might show the real value of it:
https://github.com/openshift/knative-eventing-contrib/pull/631/commits/2a5b6fe8609bb033ce46944b0695ebebbc546db6

we did (on serverless operator and knative-kafka operator) do this by hand.

Comment (aliok) [reply openshift/release/resolve.sh:23]: right. I did read this is a param for `ko` :D

**Comment by aliok** [comment]
Changes look good. Haven't really verified locally but CI will do that verification.

/lgtm
/approve

File: openshift/e2e-common.sh [hunk 2]
```diff
@@ -171,25 +171,41 @@ function install_knative_eventing(){
...
 }
 
-function install_knative_kafka(){
+function install_knative_kafka {
+  install_knative_kafka_channel
+  install_knative_kafka_source
+}
+
+function install_knative_kafka_channel(){
   header "Installing Knative Kafka components"
```
Comment (aliok) [inline openshift/e2e-common.sh:180]: maybe change the text to indicate channel resources

File: openshift/e2e-common.sh [hunk 3]
```diff
@@ -171,25 +171,41 @@ function install_knative_eventing(){
...
   cat ${RELEASE_YAML} \
-  | sed 's/namespace: .*/namespace: knative-eventing/' \
   | sed 's/REPLACE_WITH_CLUSTER_URL/my-cluster-kafka-bootstrap.kafka:9092/' \
   | oc apply --filename -
 
   wait_until_pods_running $EVENTING_NAMESPACE || return 1
 }
 
+function install_knative_kafka_source(){
+  header "Installing Knative Kafka components"
```
Comment (aliok) [inline openshift/e2e-common.sh:196]: maybe change the text to indicate source resources

**Review by aliok — APPROVED** [review]

File: openshift/release/resolve.sh (hunk 1 above)
Comment (matzew) [reply openshift/release/resolve.sh:23]: na, we "replace" `ko://` scheme with empty string in the line above
